portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,8 @@
-#event-end-time-warning {
-  visibility: hidden; 
+#event-end-time-warning,
+#event-duplicate-warning {
+  visibility: hidden;
```

Comment:
(Optional) I think it might be simpler to rely on bootstrap class names to control default visibility. See this article for example. So these elements can have the class name "d-none".
https://getbootstrap.com/docs/4.3/utilities/display/

Reply:
Updated. I used the "d-none" class to handle visibility.